src/main/java/com/exasol/adapter/document/documentfetcher/dynamodb/DynamodbFilterExpressionFactory.java

```diff
@@ -31,6 +31,17 @@ public DynamodbFilterExpressionFactory(final DynamodbAttributeNamePlaceholderMap
         this.valuePlaceholderMapBuilder = valuePlaceholderMapBuilder;
     }
 
+    public static boolean canConvert(final QueryPredicate queryPredicate) {
+        final DynamodbFilterExpressionFactory factory = new DynamodbFilterExpressionFactory(
+                new DynamodbAttributeNamePlaceholderMapBuilder(), new DynamodbAttributeValuePlaceholderMapBuilder());
+        try {
+            factory.buildFilterExpression(queryPredicate);
```

Comment:
I somehow do not like this side-effect. Is it possible to check this without building the filter expression?